transactions/src/main/java/org/jboss/as/txn/subsystem/Namespace.java

```diff
@@ -44,12 +44,13 @@
     TRANSACTIONS_3_0("urn:jboss:domain:transactions:3.0"),
     TRANSACTIONS_4_0("urn:jboss:domain:transactions:4.0"),
     TRANSACTIONS_5_0("urn:jboss:domain:transactions:5.0"),
+    TRANSACTIONS_5_1("urn:jboss:domain:transactions:5.1"),
```

Comment:
As the xml has not changed, you don't need a new xsd (it is ok for the xsd and model versions to diverge, we have other subsystems doing that)

Reply:
@ochaloup I think it looks fine now apart from this namespace is not needed

Reply:
fixed now